roles/openshift_health_checker/openshift_checks/package_availability.py

```diff
@@ -1,9 +1,9 @@
 # pylint: disable=missing-docstring
 from openshift_checks import OpenShiftCheck, get_var
-from openshift_checks.mixins import NotContainerizedMixin
+from openshift_checks.mixins import NotContainerizedMixin, NeedsYumMixin
 
 
-class PackageAvailability(NotContainerizedMixin, OpenShiftCheck):
+class PackageAvailability(NotContainerizedMixin, NeedsYumMixin, OpenShiftCheck):
```

Comment:
On Fedora you can technically still use `yum` but instead of ignoring the check shouldn't you just use `dnf`?